DisputeResolution.md

```diff
@@ -23,10 +23,15 @@ A disciplinary investigation can be started where one of the following apply:
 * all other mediation or resolution attempts have not worked
 
 ## Get advice from an Ombudsperson
-An Ombudsperson can advise you on: 
-* how the Code of Ethics and Professional Conduct may apply in your situation
-* ways to get help with mediation
-* how to raise a formal complaint
+An Ombudsperson is a sounding board who might be able to advise you about next steps with your issue. They might advise you about whether to escalate your concerns. An ombudsperson can:  

```

Comment:
I think "can advise you on" is better than "is a sounding board", because "is a sounding board" sounds like it is defining the role of the Ombud, and I don't think this document is intending to define the role of Ombud, whereas "can advise you on" is merely listing one of the things that the Ombud can do.

However, I am also confused by this document in general, because it portrays the Ombud role as a purely advisory role, whereas I had the impression -- perhaps wrongly -- that an Ombud has an element of authority and decision making power.  If an Ombud doesn't have any authority or decision making power, who does?  Only W3M?  For example, the beginning says: "In exceptional circumstances, individuals may have their W3C participation suspended or withdrawn as a result of a complaint or investigation".  But who exactly has the authority to do that, and how?